src/catkin_pkg/changelog.py

```diff
@@ -53,6 +53,8 @@
 import pkg_resources
 import re
 
+from io import open
```

Comment:
This import is required so that the encoding kwarg is accepted in both python3 and python2.